src/frontend_action.cpp

```diff
@@ -17,5 +23,34 @@ std::unique_ptr<clang::ASTConsumer> chimera::FrontendAction::CreateASTConsumer(
     is_first = false;
 
     CI.getPreprocessor().getDiagnostics().setIgnoreAllWarnings(true);
-    return std::unique_ptr<chimera::Consumer>(new chimera::Consumer(&CI));
+    return std::unique_ptr<chimera::Consumer>(
+        new chimera::Consumer(&CI, config_));
+}
+
+// Create a custom action factory that forwards the ChimeraConfiguration.
+chimera::ChimeraFrontendActionFactory::ChimeraFrontendActionFactory(
+    const chimera::Configuration &config)
+  : config_(config)
+{
+    // Do nothing.
+}
+
+#if LLVM_VERSION_AT_LEAST(8, 0, 0)
```

Comment:
This version code might be slightly wrong.  Somewhere between llvm-6 and llvm-10, this interface changed from returning a raw pointer to returning a `std::unique_ptr`.